### DISCUSSION TITLE
Complile Image without matter support

### DISCUSSION THREAD

**Discussion by kroon040** [arendst/Tasmota#19177] [Q&A]
Hi,

I try to compile an tasmota image (solo1) without matter support but with shutter support.

I have this in the platformio_tasmota_cenv.ini

```
[env:tasmota32-soloshutter]
extends                 = env:tasmota32_base
board                   = esp32_solo1
build_flags             = ${env:tasmota32_base.build_flags}
                          -DOTA_URL='"-"'
                          -DCODE_IMAGE_STR='"soloshutter"'
```
I also have this added in the  user_config_override.h

```
#ifndef USE_SHUTTER
#define USE_SHUTTER             // Add Shutter support for up to 4 shutter with different motortypes (+6k code)
#endif
```
I compile  like this

platformio run -e tasmota32-soloshutter

CXompiling works, the image is also nice small, but there is no shutter support. Looking at the image before adding something in de user_config_override.h and after this, the size still remains the same. What am I doing wrong?

**Comment by sfromis** [comment]
Where in `user_config_overide.h` did you add it? It happens that the instructions near the end are not followed.

**Comment by kroon040** [comment]
First place it near het end. Now also placed it near the lines user_config_override_h_. I added it not with nano in linux and copy the above lines and type the USE_SHUTTER

```
#ifndef _USER_CONFIG_OVERRIDE_H_
#define _USER_CONFIG_OVERRIDE_H_

#ifndef USE_SHUTTER
#define USE_SHUTTER
#endif
```
